docs/source/conf.py

```diff
@@ -191,4 +191,7 @@
 notfound_urls_prefix = None
 
 
-linkcheck_ignore = ["https://neuromorpho.org/"]
+linkcheck_ignore = [
+    "https://neuromorpho.org/",
+    "https://brainglobe.zulipchat.com/#narrow/stream/414089-developer-meeting"
+    ]
```

Comment:
```suggestion
    ]

```

pre-commit gets annoyed at me without this